core/src/main/scala/org/apache/spark/util/RpcUtils.scala

```diff
@@ -32,4 +35,24 @@ object RpcUtils {
     Utils.checkHost(driverHost, "Expected hostname")
     rpcEnv.setupEndpointRef(driverActorSystemName, RpcAddress(driverHost, driverPort), name)
   }
+
+  /** Returns the configured number of times to retry connecting */
+  def numRetries(conf: SparkConf): Int = {
+    conf.getInt("spark.rpc.num.retries", 3)
+  }
+
+  /** Returns the configured number of milliseconds to wait on each retry */
+  def retryWaitMs(conf: SparkConf): Long = {
+    conf.getLong("spark.rpc.retry.wait", 3000)
```

Comment:
Since we're here, could you use the new API:

    conf.getTimeAsMs("spark.rpc.retry.wait", "3s")

Similar in other methods in this file.